scheduler/playbooks/k8s_start_setup.yml

```diff
@@ -76,3 +79,54 @@
     https_proxy: "{{ hostvars['localhost']['https_proxy'] }}"
     no_proxy: "localhost,127.0.0.1,{{ hostvars['localhost']['admin_nic_ip'] }},{{ hostvars['localhost']['omnia_config']['k8s_service_addresses'] }},{{ hostvars['localhost']['omnia_config']['k8s_pod_network_cidr'] }},{{ hostvars['localhost']['oim_hostname'] }},{{ hostvars['localhost']['domain_name'] }}"   # noqa: yaml[line-length]
     additional_no_proxy: "{{ hostvars['localhost']['user_no_proxy'] if hostvars['localhost']['no_proxy_input_status'] else '' }}"   # noqa: yaml[line-length]
+
+- name: Initiate Kubernetes installation in online mode
+  when:
+    - hostvars['127.0.0.1']['k8s_support']
+    - not hostvars[groups['kube_control_plane'][0]]['k8s_installation_status']
+    - not hostvars['localhost']['k8s_offline_install']
+
+  ansible.builtin.import_playbook: kubernetes_sigs.kubespray.cluster
+  vars:
+    kube_version: "v{{ hostvars['localhost']['k8s_version'] }}"
+    container_manager: "containerd"
+    dashboard_enabled: true
+    helm_enabled: true
+    kube_network_plugin: "{{ hostvars['localhost']['k8s_cni'] }}"
```

Comment:
we need to use omnia_config in variables as same variables present in telemetry_config.yml